lib/school_house_web/controllers/live/conference_live/index.ex

```diff
@@ -0,0 +1,70 @@
+defmodule SchoolHouseWeb.ConferenceLive.Index do
+  @moduledoc false
+
+  use SchoolHouseWeb, :live_view
+
+  alias SchoolHouse.Conferences
+
+  @impl true
+  def mount(_params, _session, socket) do
+    conferences = Conferences.list()
+    filters = default_filters()
+    countries = countries()
+
+    new_socket =
+      socket
+      |> assign(:conferences, conferences)
+      |> assign(:filters, filters)
+      |> assign(:countries, countries)
+
+    {:ok, new_socket}
+  end
+
+  @impl true
+  def handle_event("filter", params, socket) do
+    filters = update_filters(params, socket.assigns.filters)
+    conferences = query_conferences(filters)
+
+    new_socket =
+      socket
+      |> assign(:conferences, conferences)
+      |> assign(:filters, filters)
+
+    {:noreply, new_socket}
+  end
+
+  defp update_filters(%{"filters" => new_filter}, filters) do
+    atomized = for {key, val} <- new_filter, into: %{}, do: {String.to_atom(key), val}
+
+    Map.merge(filters, atomized)
```

Comment:
Is it essential that we rely on atoms here or would strings suffice?

Reply:
I removed this, because strings work!